simplefs/simplefs.go

```diff
@@ -517,6 +542,20 @@ func (k *SimpleFS) SimpleFSClose(ctx context.Context, opid keybase1.OpID) (err e
 	return err
 }
 
+// SimpleFSCancel starts to cancel op with the given opid.
+// Also remove any pending references of opid everywhere.
+func (k *SimpleFS) SimpleFSCancel(_ context.Context, opid keybase1.OpID) {
+	k.lock.Lock()
+	defer k.lock.Unlock()
+	w, ok := k.inProgress[opid]
+	if !ok {
+		return
+	}
+	delete(k.inProgress, opid)
+	delete(k.handles, opid)
+	w.cancel()
+}
```

Comment:
Cool. Is there nothing behind the `Node` in the handle that needs to get closed / cleaned up? Just the `Node` itself?